parser/src/main/java/com/linkedin/avroutil1/parser/jsonpext/JsonReaderExt.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.parser.jsonpext;
+
+import jakarta.json.JsonReader;
+
+public interface JsonReaderExt extends JsonReader {
+
+    @Override
+    JsonStructureExt read();
```

Comment:
how is this overriden API different from the one in `jakarta.json.JsonReader` ?
It also looks like `read` and `readArray` is unsupported. Why is that the case?

Reply:
returns JsonStructureExt and not JsonStructure. others unimplemented because i didnt need them (yet?)

Reply:
Ok. maybe add some doc for this?